app/src/components/main/tooltips/tooltip/withTooltip.jsx

```diff
@@ -89,7 +89,7 @@ export const withTooltip = ({ TooltipComponent, data = {} }) => (WrappedComponen
                     onMouseLeave={data.hoverable ? this.hideTooltip : null}
                   >
                     <div
-                      className={cx('tooltip-content')}
+                      className={cx('tooltip-content', data.customClassName)}
```

Comment:
Yes, because we didn’t have the opportunity to set custom styles for the tooltip, for example, in my case, we needed our own background-color and paddings. And it turns out like this if we do not pass the custom style to _withTooltip_, but hang the styles on the root div of the tooltip itself.
![Screenshot_153](https://user-images.githubusercontent.com/70880574/205868511-da4a0e53-75d4-47d6-9ffd-2f18335e7d3c.png)
as you can see the padding and it shows the background color that is set without being able to change it.
I think this small change will improve the reusability of the component

Reply:
this props styled